src/common_modules/vsts-cli-release-common/vsts/cli/vrelease/common/release.py

```diff
@@ -0,0 +1,143 @@
+    """Get the details of a release.
+    :param release_id: ID of the release.
+    :type release_id: int
+    :param open_browser: Open the release results page in your web browser.
+    :type open_browser: bool
+    :param team_instance: VSTS account or TFS collection URL. Example: https://myaccount.visualstudio.com
+    :type team_instance: str
+    :param project: Name or ID of the team project.
+    :type project: str
+    :param detect: Automatically detect instance and project. Default is "on".
+    :type detect: str
+    :rtype: :class:`<Release> <release.v4_0.models.Release>`
+    """
+    team_instance, project = resolve_instance_and_project(detect=detect,
+                                                          team_instance=team_instance,
+                                                          project=project)
+    client = get_release_client(team_instance)
+    release = client.get_release(release_id=release_id, project=project)
+    if open_browser:
+        _open_release(release, team_instance)
+    return release
+
+
+def release_list(definition_id=None, source_branch=None, team_instance=None, project=None, detect=None, top=None,
+                 status=None):
+    """List release results.
+    :param definition_id: ID of definition to list releases for.
+    :type definition_id: int
+    :param branch: Filter by releases for this branch.
+    :type branch: str
+    :param team_instance: VSTS account or TFS collection URL. Example: https://myaccount.visualstudio.com
+    :type team_instance: str
+    :param project: Name or ID of the team project.
+    :type project: str
+    :param detect: Automatically detect instance and project. Default is "on".
+    :type detect: str
+    :param top: Maximum number of releases to list. Default is 50.
+    :type top: int
+    :param status: Limit to releases with this status.
+    :type status: str
+    :param source_branch: Filter releases for this branch.
+    :type source_branch: str
+    :rtype: :class:`<Release> <release.v4_0.models.Release>`
+    """
+    team_instance, project = resolve_instance_and_project(detect=detect,
+                                                           team_instance=team_instance,
+                                                           project=project)
+    client = get_release_client(team_instance)
+
+    releases = client.get_releases(definition_id=definition_id,
+                                project=project,
+                                source_branch_filter=source_branch,
+                                top=top,
+                                status_filter=status)
+    return releases
+
+
+def _open_release(release, team_instance):
+    """Open the release results page in your web browser.
+    :param :class:`<Release> <release.v4_0.models.Release>` release:
+    :param str team_instance:
+    """
+    # https://mseng.visualstudio.com/vsts-cli/_release/index?releaseId=4053990
```

Comment:
does this url work for release?

Reply:
I have removed the hardcoding and changed the logic to get web url from release object